codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -198,9 +200,9 @@ int streamableBytesForTree(Http2Stream stream) {
         return state(stream).streamableBytesForTree();
     }
 
-    private static FlowState state(Http2Stream stream) {
+    private FlowState state(Http2Stream stream) {
```

Comment:
This is used in inner class (not that you introduced this).  Make package protected.